### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by elenatvk**
doc review of the readme

### CODE REVIEW

File: templates/sag-db-oracle/README.md [hunk 1]
```diff
@@ -21,24 +21,42 @@
...
+--------------------|----------|---------------------|------------------
+webMethods Optimize  |OBEcdc  |  db.OBE.components | schemas.OBE
+webMethods Business Rules | WOKcdc|  db.WOK.components | schemas.WOK
+webMethods Trading Networks | TNScdc  |  db.TNS.components | schemas.TNS
+My webMethods Server | MWScdc| db.MWS.components | schemas.MWS
+webMethods B2B Cloud  |  B2BcloudCdc| db.B2Bcloud.components | schemas.B2Bcloud
+webMethods Active Transfer | MATcdc | db.MAT.components | schemas.MAT
+webMethods OneData |  ODEcdc | db.ODE.components | schemas.ODE
+webMethods Integration Server and webMethods Microservices Runtime | PIEcdc | db.PIE.components | schemas.PIE
+Integration Server | PIEEmbeddedCdc | Not applicable (N/A) | N/A
```
Comment (YanaSimeonova) [inline templates/sag-db-oracle/README.md]: PIEEmbeddedCDC is special case - it has no component. i'm not sure if "not applicable" is the correct statement.

Comment (elenatvk) [reply templates/sag-db-oracle/README.md]: How about not available?

Comment (YanaSimeonova) [reply templates/sag-db-oracle/README.md]: yes. it is good

Comment (elenatvk) [reply templates/sag-db-oracle/README.md]: okay, I edited the file

**Pull request merged**